cosim/zynq-parrot-dromajo/Makefile

```diff
@@ -0,0 +1,40 @@
+TOP  := $(shell git rev-parse --show-toplevel)
```

Comment:
The FPGA variant is useful, but we should put it in the FPGA directory. The x86 one is covered by devtoolset which is our supported methodology

Reply:
Roger that